### ISSUE TITLE
Feature request: mass mailings

### ISSUE THREAD

**Issue by erdgeist** [frab/frab#234]
tl;dr: Send all mails regarding confirmation/rejection and time/room at once when conference state changes. Also have templates to generate all mails at once.

Rationale: while coordinating 24C3 - 32C3 we noticed, there are two huge time wasters for our track teams: Generating (personalized) confirmation/rejection emails and generating emails telling the speakers, when and where they speak. The second one is checking (IN)activity in tickets from frab, which will be addressed in another issue.

Also sending all confirmation/rejection emails at once resolves the awkward limbo speakers experience when other speakers report them being accepted via twitter.

What I have in mind is an attachment to an event that is automatically filled from a template (if possible matching the event's language), when this event is accepted/rejected and can then be personalized, if necessary. Once the final round of decisions on submissions for a conference is through, and all letters needing personalization are edited, a member of the orga group can then initiate the mass mailing from those text attachments (possibly warning about missing tickets on events).

From then on, all accepted/rejected talks will be sent immediately (as currently implemented).

Once the final Fahrplan is set, a similar method would be employed to auto-generate all emails from a template containing room and date information. These emails need not be sent at once.

**Comment by ScottyTM** [comment]
You are aware of https://frab.example.com/en/example16/conference/edit_notifications where you can configure the templates for the acceptance and rejection mails generated  by frab when you click on Accept event?

**Comment by manno** [comment]
There is also 'Mail Templates' http://localhost:3000/en/example23/mail_templates where you can write to all confirmed/unconfirmed speakers.

**Comment by erdgeist** [comment]
Yes, I am aware of those templates, however they currently only substitute public_name, first_name and last_name. For the second lecture detail email, at least date, time and room would need to be substituted.

**Comment by erdgeist** [comment]
Is there any particular reason for not using the http://www.rubydoc.info/gems/rt-client gem? I'd re-implement RT communication and send mails via this API, if this was an option.

**Comment by manno** [comment]
Sounds good.

**Comment by derchrisuk** [comment]
Is the https://frab.example.com/en/example16/conference/edit_notifications link only available when using the internal ticket system?
I don't have a link visible, but can open the URL when entered manually

**Comment by erdgeist** [comment]
Implemented in #268.
